ProcessLib/Parameter/MeshNodeParameter.h

```diff
@@ -57,6 +59,27 @@ struct MeshNodeParameter final : public Parameter<T> {
         return _cache;
     }
 
+    Eigen::Matrix<T, Eigen::Dynamic, Eigen::Dynamic> getNodalValuesOnElement(
+        MeshLib::Element const& element, double const t) const override
+    {
```

Comment:
Doesn't the base implementation do exactly the same for nodal params?

Reply:
There is a small difference. If I find some time to measure the differences, I'll report here.

Reply:
No need to measure. OK for me, even though the difference is only in the saved `element.getID()` essentially.